docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1007,6 +1013,18 @@ public class MyConfiguration {
 ----
 ====
 
+[[loadbalancer-lifecycle]]
+=== Spring Cloud LoadBalancer Lifecycle
+
+One type of beans that it may be useful to register using <<custom-loadbalancer-configuration,Custom LoadBalancer configuration>> is `LoadBalancerLifecycle`.
+
+The `LoadBalancerLifecycle` beans provide callback `onStart(Request<RC> request)` and `onComplete(CompletionContext<RES, T> completionContext)` methods that you should implement to specify what actions should take place before and after load-balancing.
```

Comment:
Change to

The `LoadBalancerLifecycle` beans provide callback methods, named `onStart(Request<RC> request)` and `onComplete(CompletionContext<RES, T> completionContext)`, that you should implement to specify what actions should take place before and after load-balancing.